### ISSUE TITLE
Release notes is empty on v1.3.3

### ISSUE THREAD

**Issue by dashika** [wzieba/Firebase-Distribution-Github-Action#67]
I updated the version from 1.3.2 to 1.3.3 and stopped filling release notes, it is empty. Can you help me with this, maybe in new version need some configuration?

**Comment by wzieba** [comment]
Thanks for the report - it shouldn't require any changed configuration so it's rather a bug in this action, sorry.

I'll try to work on this ASAP.

**Comment by wzieba** [comment]
But looking at changes between 1.3.2 and 1.3.3 I can't see a reason to break release notes. Maybe Firebase CLI changed their API, I have to check that.

Do you use `releaseNotes` or `releaseNotesFile` to pass release notes?

**Comment by dashika** [comment]
No, I don't use releaseNotes and releaseNotesFile.

**Comment by brian-pinwheel** [comment]
I have the same problem that release notes have shown up as empty.  I use the plugin for two builds.  The one where I specify `releaseNotes` is working as expected.  The one that I don't specify it and default to the commit hash code,  the release notes on Firebase are empty.  

I agree that it might be an issue with changing APIs somewhere.

**Comment by yoshuaandrean** [comment]
any updates on this?

**Comment by SergeiMikhailovskii** [comment]
@dashika @yoshuaandrean 

as a workaround, you can use this script:

```
  distribute:
    needs: build
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: actions/download-artifact@v3
        with:
          name: androidApp-debug.apk

      - name: Get latest commit
        id: vars
        run: echo ::set-output name=LATEST_COMMIT_MESSAGE::$(echo $(git log -1 --pretty=format:"%s"))

      - name: Upload artifact to FAD
        uses: wzieba/Firebase-Distribution-Github-Action@v1
        with:
          appId: ${{secrets.FIREBASE_APP_ID}}
          token: ${{secrets.FIREBASE_TOKEN}}
          groups: testers
          file: androidApp-debug.apk
          releaseNotes: ${{ steps.vars.outputs.LATEST_COMMIT_MESSAGE }}
```

Here latest commit is fetched manually and saved into variable. then the release notes are taken from the variable

**Comment by dashika** [comment]
Works, thanks.

**Comment by brian-pinwheel** [comment]
> @dashika @yoshuaandrean
> 
> as a workaround, you can use this script:
> 
> ```
>   distribute:
>     needs: build
>     runs-on: ubuntu-latest
>     steps:
>       - uses: actions/checkout@v2
>       - uses: actions/download-artifact@v3
>         with:
>           name: androidApp-debug.apk
> 
>       - name: Get latest commit
>         id: vars
>         run: echo ::set-output name=LATEST_COMMIT_MESSAGE::$(echo $(git log -1 --pretty=format:"%s"))
> 
>       - name: Upload artifact to FAD
>         uses: wzieba/Firebase-Distribution-Github-Action@v1
>         with:
>           appId: ${{secrets.FIREBASE_APP_ID}}
>           token: ${{secrets.FIREBASE_TOKEN}}
>           groups: testers
>           file: androidApp-debug.apk
>           releaseNotes: ${{ steps.vars.outputs.LATEST_COMMIT_MESSAGE }}
> ```
> 
> Here latest commit is fetched manually and saved into variable. then the release notes are taken from the variable

Thanks @wzieba for fixing the issue!  Since this was a regression with the 1.3.3 release, any chance there might be a 1.3.4 release with this fix in the near future?

**Comment by wzieba** [comment]
hi @brian-pinwheel 👋 

I did not fix the issue, the hotfix provided by @SergeiMikhailovskii requires adding another GA step.

But the PR #77 provided by @davidmigloz seems to fix the issue:

<img width="528" alt="Screenshot 2022-06-20 at 15 47 55" src="https://user-images.githubusercontent.com/5845095/174615954-2b33d7dd-4f6c-4a26-8946-741335093ae1.png">

You (or anyone interested) can try the fix using:

```
wzieba/Firebase-Distribution-Github-Action@9bc1d48a99c3447060a8facc22f078e1e40ed464
```

If the above works for you well, I'll prepare the fix. Thank you!

**Comment by brian-pinwheel** [comment]
@wzieba Fixed it!  

I was looking at the FB site and it appears the commits are rendering a bit different in Firebase (maybe they changed something), but went back to the previous build and found those release notes were rendering differently too.  I mention it in case that is a permanent change and it catches people off guard.

Thanks @SergeiMikhailovskii for fixing it and @wzieba for continuing to curate a very help open source project!  

Any approximate timeframe for a new release?

**Comment by wzieba** [comment]
No problem @brian-pinwheel! I've just released `1.3.4` with the fix: https://github.com/wzieba/Firebase-Distribution-Github-Action/releases/tag/v1.3.4

**Comment by brian-pinwheel** [comment]
Thanks @wzieba!!  I know from personal experience is a relatively thankless job but do appreciate the amount of work it involves.  I appreciate not only creating a useful library but also actively maintaining it!
